src/main/scala/com/amazon/deequ/analyzers/CustomSql.scala

```diff
@@ -26,6 +26,17 @@ import scala.util.Failure
 import scala.util.Success
 import scala.util.Try
 
+case class CustomSqlState(stateOrError: Either[Double, String]) extends DoubleValuedState[CustomSqlState] {
```

Comment:
Either is right biased in Scala, which means the right sided value is what you would operate on when doing `map` or `flatMap`. In the case of storing exceptions or error messages, usually they are stored as the left sided value of the Either.

Reply:
I'm just moving this from Size.scala to its own class, I can address that in a different PR